coderd/database/queries/provisionerjobs.sql

```diff
@@ -90,7 +90,7 @@ final_jobs AS (
 	FROM
 		filtered_provisioner_jobs fpj -- Use the pre-filtered dataset instead of full provisioner_jobs
 			LEFT JOIN ranked_jobs rj
-					ON fpj.id = rj.id -- Join with the ranking jobs CTE to assign a rank to each specified provisioner job.
+				 ON fpj.id = rj.id -- Join with the ranking jobs CTE to assign a rank to each specified provisioner job.
```

Comment:
Nit: We don't have strict guidelines, but this is an unconventional indentation, we don't align keywords.